src/app/docs/avatar/avatar-docs.component.html

```diff
@@ -0,0 +1,20 @@
+<sky-docs-demo-page
+  moduleName="SkyAvatarModule"
+  moduleSourceCodePath="src/app/public/modules/avatar/"
+  pageTitle="Avatar"
+>
+  <sky-docs-demo-page-summary>
+    The avatar component displays an image to identify a record and includes an option to let users change the image. If the image is missing, the avatar component can display initials for the record that the avatar represents.
```

Comment:
I'd cut this down to just `The avatar component displays an image to identify a record.` We've been trying to shorten the overviews in phase 2 docs, and the rest of the info here should be covered in the Options section when we create the Design tab guidelines.

Reply:
Updated.